cmd/infra/aws/destroy.go

```diff
@@ -100,10 +100,10 @@ func (o *DestroyInfraOptions) DestroyInfra(ctx context.Context) error {
 
 	var errs []error
 	errs = append(errs, o.DestroyInternetGateways(ctx, ec2Client)...)
+	errs = append(errs, o.DestroyDNS(ctx, route53Client)...)
 	errs = append(errs, o.DestroyVPCs(ctx, ec2Client, elbClient, route53Client)...)
```

Comment:
I almost think that we should put DestroyVPCs in a separate loop after all the reset since it is the foundation for the deprovisioner to get a hook on anything that is left over.